packages/scandipwa/src/component/SearchOverlay/SearchOverlay.component.tsx

```diff
@@ -80,22 +88,51 @@ export class SearchOverlayComponent extends PureComponent<SearchOverlayComponent
         );
     }
 
+    renderSearchOverlayResults(): ReactElement {
+        const { activeClosingAnimation } = this.state;
+
+        return (
+            <div
+              block="SearchOverlay"
+              elem="Results"
+              aria-label="Search results"
+              mods={ { activeClosingAnimation } }
```

Comment:
`mods` should follow the `isSomeValue` style, so it would be `isActiveClosingAnimation`